### PR TITLE
indicates that the message is sent via wa web automatically

### PR DESCRIPTION
**Pull request by RC047**
# PR Details
Added the option that the message is sent via whatsapp web

## Description
Some of these possibilities can be useful to be able to distinguish which messages are sent via whatsApp web automatically and which are not

## Motivation and Context
So that now it can detect which messages are sent via wa web automatically and which are not

## How Has This Been Tested
I've tested it with my own Bot on windows platform and it works great!

## Types of changes

- [ ] Dependency change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

File: src/structures/Message.js [hunk 1]
```diff
@@ -129,6 +129,12 @@ class Message extends Base {
          */
         this.fromMe = data.id.fromMe;
 
+        /**
+         * Indicates if the message was sent via whatsapp web
+         * @type {boolean}
+         */
+        this.isWWebJS = data.id.id.startsWith('3EB0') && data.id.id.length === 12;
```
Comment (alechkos) [inline src/structures/Message.js:136]: What about [`this.deviceType`](https://github.com/pedroslopez/whatsapp-web.js/blob/8f5d9ad6fce6a8e8746176892adc0b796d96e1d6/src/structures/Message.js#L93)?

Comment (RC047) [reply src/structures/Message.js:136]: `this.deviceType` is only to indicate the type of device that is connected to whatsapp web, this one's function is to distinguish which messages are sent automatically and which messages are sent manually in whatsapp web

**Comment by tuyuribr** [comment]
Totally a bad Idea

**Comment by renzhxr** [comment]
is this a solution so that bots respond to commands sent from wa web?

**Comment by RC047** [comment]
> is this a solution so that bots respond to commands sent from wa web?

yes

**Comment by aalwhaib** [comment]
On event "message_created" just check if `message.ack === 0 `
this mean the message send automatically

**Pull request closed without merging**